bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -674,6 +674,8 @@ private void updateReviewedMessages(List<Comment> comments, List<Review> reviews
 
         for (var added : reviewTracker.newReviews().entrySet()) {
             var body = added.getValue() + "\n" +
+                    " ⚠️ Marking a PR as reviewed using the emoji buttons is deprecated and will not be supported in the near future. " +
+                    "Please use the `Approve` button instead!\n\n" +
```

Comment:
Not sure whether we need the exclamation mark or not, I leave that up to you :smile_cat:

Reply:
No idea, but can't hurt I guess.. :)